src/engine/platform/x1_010.cpp

```diff
@@ -942,6 +942,11 @@ void DivPlatformX1_010::setFlags(const DivConfig& flags) {
   CHECK_CUSTOM_CLOCK;
   rate=chipClock/512;
   stereo=flags.getBool("stereo",false);
+  bool prevBanked=isBanked;
+  isBanked=flags.getBool("isBanked",false);
+  if (prevBanked!=isBanked) {
+    parent->renderSamples();
```

Comment:
Tbis is absolutely not a good idea. If you need to call renderSamples(), call the dispatch's own version, but not the global one.